src/ru/mail/track/UserStorage.java

```diff
@@ -0,0 +1,69 @@
+package ru.mail.track;
+
+import java.io.*;
```

Comment:
Preferences->Editor->CodeStyle->Java
Import tab
set class count to use '*' = 20